src/desktop/settings.rs

```diff
@@ -232,9 +232,7 @@ impl<'a> Settings<'a> {
     }
 
     /// Listen to changes of the system's accent color
-    pub async fn receive_accent_color_changed(
-        &self,
-    ) -> Result<impl Stream<Item = ColorScheme>, Error> {
+    pub async fn receive_accent_color_changed(&self) -> Result<impl Stream<Item = Color>, Error> {
```

Comment:
whats the justification of this change?

Reply:
The `ColorScheme` type is not the correct type. I explained in the commit that changes it that it just seems like a typo. The accent-color value is a color. `ColorScheme` is meant for the color-scheme value.
```
pub enum ColorScheme {
    NoPreference,
    PreferDark,
    PreferLight,
}
```

Reply:
Sorry, i didn't get that the `Item = Color` was different in the diff, move along.